code/modules/admin/tickets/adminticketsUI.dm

```diff
@@ -0,0 +1,150 @@
+/datum/adminTicketHolder/proc/returnUI(var/tab = ADMIN_TICKET_OPEN)
```

Comment:
Same

Reply:
This was addressed in the latest commit.